qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -34,29 +37,34 @@ class QSVC(SVC):
 
     .. code-block::
 
-        qsvc = QSVC(quantum_kernel=qkernel)
+        qsvc = QSVC(quantum_kernel=quant_kernel)
         qsvc.fit(sample_train,label_train)
         qsvc.predict(sample_test)
     """
 
-    def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwargs):
+    def __init__(
+        self,
+        *args,
+        quantum_kernel: Optional[QuantumKernel] = None,
```

Comment:
From what I can see this needs to be Optional[Union[QuantumKernel, QuantumKernelTrainer]] given what I see below